index.bs

```diff
@@ -721,28 +724,24 @@ spec:dom-ls; type:interface; text:Document
   The values of <a link-type="grammar">suborigin-policy-option</a> that may be
   present in a <a>suborigin policy</a> have the following effects:
 
-  * '<dfn>`unsafe-postmessage-receive`</dfn>' When a message is sent <i>to</i> a
```

Comment:
Ah, yeah, did two things at once. I'll undo that and redo it separately if I feel like it's needed.